api/v1/cluster_webhook.go

```diff
@@ -769,9 +775,32 @@ func (r *Cluster) validateConfigurationChange(old *Cluster) field.ErrorList {
 		}
 	}
 
+	if err := validateSyncReplicaElectionConstraint(
+		r.Spec.PostgresConfiguration.SyncReplicaElectionConstraint,
+	); err != nil {
+		result = append(result, err)
+	}
+
```

Comment:
This should not be needed, we are already checking the same thing in the Validate method, which is called before ValidateChange on updates.

Reply:
```suggestion
```